README.md

```diff
@@ -86,19 +86,6 @@ return {
 
     -- see below for full list of options 👇
   },
-  config = function(_, opts)
-    require("obsidian").setup(opts)
-
-    -- Optional, override the 'gf' keymap to utilize obsidian.nvim's search functionality.
-    -- see also: 'follow_url_func' config option below.
-    vim.keymap.set("n", "gf", function()
```

Comment:
In the spirit of updating the docs, we might want to leave this information somewhere in the README in to provide some direction for users who wish to manage their commands manually. For example, I don't think we document anywhere that this helper function `cursor_on_markdown_link()` exists, which might be very helpful information for power users.

Reply:
Yeup, agreed: 3da669d